src/index.ts

```diff
@@ -21,6 +21,16 @@ interface Config {
   version?: string;
 }
 
+/**
+ * Replaces characters not allowed in Cloudinary parameter values
+ */
+function cleanText(text: string): string {
+  return text
```

Comment:
it would make sense to pull the `encodeURIComponent` call into this util, since it's part of the same thing

```suggestion
  return encodeURIComponent(text)
```

Reply:
good idea, done!